sklearn/utils/estimator_checks.py

```diff
@@ -273,7 +273,6 @@ def _yield_transformer_checks(transformer):
         "Isomap",
         "KernelPCA",
         "LocallyLinearEmbedding",
-        "RandomizedLasso",
```

Comment:
What is `RandomizedLasso`?

Reply:
Apparently it existed at some point int he past lol